### DISCUSSION TITLE
why?

### DISCUSSION THREAD

**Discussion by Assassinator-ban** [flet-dev/flet#4532] [General]
flet build apk
[01:56:38] Created Flutter bootstrap project from                               
           gh:flet-dev/flet-build-template with ref 0.25.1 ✅                   
           Customized app icons and splash images ✅                            
[01:57:50] Generated app icons ✅                                               
[01:57:53] Generated splash screens ✅                                          
[01:58:32] Packaged Python app ✅                                               
[01:58:35] Resolving dependencies...                                            
           Downloading packages...                                              
             archive 3.6.1 (4.0.1 available)                                    
             characters 1.3.0 (1.3.1 available)                                 
             clock 1.1.1 (1.1.2 available)                                      
             collection 1.18.0 (1.19.1 available)                               
             flutter_lints 2.0.3 (5.0.0 available)                              
             http_parser 4.0.2 (4.1.1 available)                                
             intl 0.19.0 (0.20.1 available)                                     
             js 0.6.7 (0.7.1 available)                                         
             lints 2.1.1 (5.1.0 available)                                      
             material_color_utilities 0.11.1 (0.12.0 available)                 
             meta 1.15.0 (1.16.0 available)                                     
             path 1.9.0 (1.9.1 available)                                       
             sensors_plus 4.0.2 (6.1.0 available)                               
             sensors_plus_platform_interface 1.2.0 (2.0.0 available)            
             serious_python 0.8.3 (0.8.4 available)                             
             shelf 1.4.1 (1.4.2 available)                                      
             toml 0.15.0 (0.16.0 available)                                     
             url_strategy 0.2.0 (discontinued)                                  
           ! wakelock_plus 1.2.8 (overridden)                                   
           ! web 1.1.0 (overridden)                                             
             web_socket_channel 2.4.5 (3.0.1 available)                         
           ! window_manager 0.4.3 (overridden)                                  
           Got dependencies!                                                    
           1 package is discontinued.                                           
           19 packages have newer versions incompatible with dependency         
           constraints.                                                         
           Try `flutter pub outdated` for more information.                     
                                                                                
           Running Gradle task 'assembleRelease'...                             
           Running Gradle task 'assembleRelease'...                             
           121ms                                                                
                                                                                
           Exception in thread "main" java.util.zip.ZipException: zip END header
           not found                                                            
                   at java.base/java.util.zip.ZipFile$Source.findEND(Unknown    
           Source)                                                              
                   at java.base/java.util.zip.ZipFile$Source.initCEN(Unknown    
           Source)                                                              
                   at java.base/java.util.zip.ZipFile$Source.<init>(Unknown     
           Source)                                                              
                   at java.base/java.util.zip.ZipFile$Source.get(Unknown Source)
                   at                                                           
           java.base/java.util.zip.ZipFile$CleanableResource.<init>(Unknown     
           Source)                                                              
                   at java.base/java.util.zip.ZipFile.<init>(Unknown Source)    
                   at java.base/java.util.zip.ZipFile.<init>(Unknown Source)    
                   at java.base/java.util.zip.ZipFile.<init>(Unknown Source)    
                   at org.gradle.wrapper.Install.unzip(Install.java:214)        
                   at org.gradle.wrapper.Install.access$600(Install.java:27)    
                   at org.gradle.wrapper.Install$1.call(Install.java:74)        
                   at org.gradle.wrapper.Install$1.call(Install.java:48)        
                   at                                                           
           org.gradle.wrapper.ExclusiveFileAccessManager.access(ExclusiveFileAcc
           essManager.java:65)                                                  
                   at org.gradle.wrapper.Install.createDist(Install.java:48)    
                   at                                                           
           org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:128) 
                   at                                                           
           org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61) 
           Gradle task assembleRelease failed with exit code 1                  
                                                                                
           Error building Flet app - see the log of failed command above.       
[01:58:36] Doctor summary (to see all details, run flutter doctor -v):          
           [✓] Flutter (Channel stable, 3.24.5, on Ubuntu 24.04.1 LTS           
           6.8.0-49-generic, locale zh_CN.UTF-8)                                
           [✓] Android toolchain - develop for Android devices (Android SDK     
           version 34.0.0)                                                      
           [✓] Chrome - develop for the web                                     
           [✓] Linux toolchain - develop for Linux desktop                      
           [✓] Android Studio (version 2024.2)                                  
           [✓] VS Code (version 1.95.3)                                         
           [!] Proxy Configuration                                              
               ! NO_PROXY does not contain 127.0.0.1                            
           [✓] Connected device (2 available)                                   
           [✓] Network resources                                                
                                                                                
           ! Doctor found issues in 1 category.

**Comment by Assassinator-ban** [comment]
![Snipaste_2024-12-10_02-01-49](https://github.com/user-attachments/assets/6a31d7f7-6968-4ae1-be86-98488ef0981f)

**Comment by Assassinator-ban** [comment]
![Uploading Snipaste_2024-12-10_02-01-49.png…]()

**Comment by ndonkoHenri** [comment]
Run with --v flag and share output.

Also, I can't view the image you sent.
